Model/Animal.cs

```diff
@@ -0,0 +1,73 @@
+﻿    public class Animal
+    {
+    internal const ushort MAX_AGE = 2000;
+    internal const float MIN_SIZE = 0.03f;
+    internal const float MAX_SIZE = 33.0f;
+    internal const float MAX_JUMP_LENGTH = 5.25f;
+    internal  const byte MAX_FROG_AGE = 17;
+    internal  const float MIN_FROG_SIZE = 0.008f;
+    internal const float MAX_FROG_SIZE = 0.07f;
+    internal const byte MAX_CAT_AGE = 43;
+    internal const float MIN_CAT_SIZE = 0.08f;
+    internal const float MAX_CAT_SIZE = 0.55f;
+    internal string _name="default name";
+    internal ushort _age=0;
+    internal float _size= MIN_SIZE;
+    public Animal()
+    {
+    }
+    public Animal(string name)
+    {
+        Name = name;
+    }
+    public Animal(string name, ushort age, float size)
+    {
+        Name = name;
+        Age = age;
+        Size = size;
+    }
+    //properties
+    public string Name
+        {
+            get { return _name; }
+            set { if( _name != null)
```

Comment:
только проверять надо value при set'e
```suggestion
            set { if( value != null)
```